### PR TITLE
made text/css changes to ground-failure pin

### LINKED ISSUE #1022: Adjust wording on ground failure pin/card
Add the word "affected" to the end of the hazard short sentences and "exposed" to the end of each short sentence on the product pin (e.g., Significant area affected, Limited population exposed). This is experimental, we received this suggestion and like the idea but aren't sure how it will look on the card, so we want to see how it looks on the webpage before deciding for sure one way or the other.

### PR DESCRIPTION
**Pull request by cswilson-usgs**
Fixes #1022

### CODE REVIEW

File: src/app/executive/basic-pin/basic-pin.component.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@ section {
   font-size: 0.88em;
   margin: 0.5em 0;
   max-height: 100%;
-  padding: 0 0.75em;
+  padding: 0 0.75em;
```
Comment (ehunter-usgs) [inline src/app/executive/basic-pin/basic-pin.component.scss]: trailing whitespace

File: src/app/oaf/forecast/forecast.component.html [hunk 2]
```diff
@@ -6,6 +6,11 @@
     period. These longer periods may be useful when planning recovery and
     rebuliding projects.
   </p>
+  
+  <hr>
+    <h1>TEST!</h1>
+    {{ forecast.rows | json }}
+  <hr>
```
Comment (ehunter-usgs) [inline src/app/oaf/forecast/forecast.component.html]: Remove this code.

File: src/app/executive/ground-failure-pin/ground-failure-pin.component.html [hunk 3]
```diff
@@ -3,105 +3,115 @@
   [product]="product"
   [title]="title">
 
-  <div>
```
Comment (ehunter-usgs) [inline src/app/executive/ground-failure-pin/ground-failure-pin.component.html]: After you deleted the wrapper div you didn't outdent all the content.

**Comment by emartinez-usgs** [comment]
Please remove unused import.

**Pull request merged**